### ISSUE TITLE
test_semisup gives Validation error rate around 0.12 for svhn after training for 100 epochs

### ISSUE THREAD

**Issue by pratikm141** [9310gaurav/ali-pytorch#1]
Which configuration is this referring to in the original paper for SVHN test set misclassification rate. 
Is it:
1) ALI (ours, L2-SVM) ----->19.14 ± 0.50 (in paper)

or
 
2) ALI (ours, no feature matching) ----->7.42 ± 0.65 (in paper)

netE_90 provided by author has Validation error rate = 0.1583 +- 0.0

**Comment by 9310gaurav** [comment]
This implementation refers to 1. where the model is trained in an unsupervised way.

**Comment by pratikm141** [comment]
Ok so is it normal that I am getting 12% on the same code?

**Comment by 9310gaurav** [comment]
It is quite surprising. The saved model that I got after training for 100 epochs gave 23% and I tried many runs. Did you make any changes at all?

**Comment by pratikm141** [comment]
No I have not made any changes.
Infact I am getting around 15% for the netE_epoch_90.pth provided by default in this repository using the test_semisup.py provided in the repository.
I am attaching the netE_epoch_90.pth that I got after training on SVHN for which i got 13.4% validation error. 
[netE_epoch_90.zip](https://github.com/9310gaurav/ali-pytorch/files/3487648/netE_epoch_90.zip)

for epoch 100 I got around 12% validation error

**Comment by pratikm141** [comment]
Can you please check the validation error for my saved model.

**Comment by 9310gaurav** [comment]
I am a bit busy this week. I will run your attached model and my model as well this weekend and let you know.

**Comment by pratikm141** [comment]
ok thanks

**Comment by pratikm141** [comment]
Hi, did u check the models?
